### DISCUSSION TITLE
How to delete configuration from other devices

### DISCUSSION THREAD

**Discussion by slimjet4u** [vrtmrz/obsidian-livesync#448] [Q&A]
I am using livesync on two devices mobile and windows, Whenever I delete the vault from mobile and use the uri generated from pc, somehow the mobile device still remembers the previous devices in the custom sync. I still see devices that I had used before. 

How to reset everything from all the device ??

**Comment by vrtmrz** [comment]
Thank you for your question! If you are referring to the device names in each drop-down, they are stored in the remote database.
You can delete all of them by the `Delete All of ...` tool if enabling `Maintenance mode` in the `Maintenance command`.

![image](https://github.com/vrtmrz/obsidian-livesync/assets/45774780/f7a3cc74-07e6-4d5a-8b29-5263c363ab01)

If you have mentioned automatic or selective toggle (as below), they will be overwritten by the normal setup-URI.
![image](https://github.com/vrtmrz/obsidian-livesync/assets/45774780/94e595cc-cb01-46d3-9337-238d8367019c)
